scalyr_agent/log_processing.py

```diff
@@ -2889,6 +2905,7 @@ def __rename_log_file(self, matched_file, log_config):
                     result = pattern.substitute(values)
                 except Exception as e:
                     log.warn(
+                        # 2->TODO same as 2046 line.
```

Comment:
Line numbers can change... just repeat the comment.

Reply:
Yes, that was a bad idea...